rocAL/include/api/rocal_api_tensor.h

```diff
@@ -40,6 +40,7 @@ class rocalTensor {
     virtual ~rocalTensor() = default;
     virtual void* buffer() = 0;
     virtual unsigned copy_data(void* user_buffer, RocalOutputMemType external_mem_type = ROCAL_MEMCPY_HOST) = 0;
+    virtual unsigned copy_data(void* user_buffer, uint max_cols, uint max_rows) = 0; // Copy only the ROI to the user_buffer [The padded region is not copied]
```

Comment:
why mem_type is removed?
instead of max_cols and max_rows, I prefer passing an roi with (xoff, yoff, w, h)